app/middlewares/user.js

```diff
@@ -0,0 +1,36 @@
+const { checkSchema, validationResult } = require('express-validator');
+
+exports.dataUser = (req, res, next) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(422).json({ errors: errors.mapped() });
+  }
+  return next();
+};
```

Comment:
This validation could be not only for users, it could be used for any entity and for any endpoint; What do you think if you put this logic in a separate file, inside the middlewares and make it call something like `validationSchema`